docs/control_panel_integration/README.md

```diff
@@ -77,9 +77,15 @@ That means that you can add `Conficts: public_cp_vendors_api < VERSION` to the s
 
 ### Changelog
 ::: tip Changelog
+
+Version 1.2
+1. `Provides public_cp_vendors_api = 1.2` added to rpm spec of alt-python27-cllib package (see [versioning](/control_panel_integration/#versioning)).
+2. Possibilty to run [`domains` integration script](https://docs.cloudlinux.com/control_panel_integration/#domains) with optional flag `--with-php` added.
```

Comment:
Changed [`domains` integration script](https://docs.cloudlinux.com/control_panel_integration/#domains). Added new optional flag `--with-php` that is required to integrate x-ray.

Reply:
Updated in 3rd commit.